flutter-studio/src/io/flutter/project/FlutterProjectSystem.java

```diff
@@ -102,27 +102,19 @@ public ProjectSystemSyncManager getSyncManager() {
     }
   }
 
-  @SuppressWarnings("override")
-  public boolean getAugmentRClasses() {
-    // TODO(messick) Remove when 3.5 is stable
-    return false;
-  }
-
   @NotNull
   @SuppressWarnings("override")
   public LightResourceClassService getLightResourceClassService() {
     return gradleProjectSystem.getLightResourceClassService();
   }
 
-  @NotNull
-  @SuppressWarnings("override")
-  public SourceProvidersFactory getSourceProvidersFactory() {
-    return gradleProjectSystem.getSourceProvidersFactory();
-  }
+  //@NotNull
+  //public SourceProvidersFactory getSourceProvidersFactory() {
+  //  return gradleProjectSystem.getSourceProvidersFactory();
+  //} // for 4.0
 
-  @NotNull
-  @SuppressWarnings("override")
-  public Collection<Module> getSubmodules() {
-    return gradleProjectSystem.getSubmodules();
-  }
+  //@NotNull
+  //public Collection<Module> getSubmodules() {
+  //  return gradleProjectSystem.getSubmodules();
+  //}
```

Comment:
does this need "// for 4.0"

Reply:
It is not used. I didn't delete it because I think it will be needed in a later version.